### PR TITLE
New recipe for Meraculous.

### PR DESCRIPTION
**Pull request by ysu20**
* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [x] This PR adds a new recipe.
* [x] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

**Comment by ysu20** [comment]
This is my first PR to this project.  Review and feedback are welcome.

File: recipes/meraculous/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    md5: 349feb6cb178643a46e4b092c87bad3a
+
+build:
+    skip: True  # [osx]
+    number: 0
+
+requirements:
+    build:
+    - cmake >=2.8
+    - gcc ==4.8.5 # [linux]
```
Comment (bgruening) [inline recipes/meraculous/meta.yaml]: you don't need to pin gcc here

File: recipes/meraculous/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+    - libgd >=2.0
+    - boost-cpp ==1.65.0
+    - perl >=5.10
+    - perl-log-log4perl >=1.31
+    - gnuplot >=3.7
+
+about:
+    home: https://jgi.doe.gov/data-and-tools/meraculous/
+    summary: Meraculous is a whole genome assembler for Next Generation Sequencing data, geared for large genomes. It's hybrid k-mer/read-based approach capitalizes on the high accuracy of Illumina sequence by eschewing an explicit error correction step which we argue to be redundant with the assembly process. Meraculous achieves high performance with large datasets by utilizing lightweight data structures and multi-threaded parallelization, allowing to assemble human-sized genomes on a high-cpu cluster in under a day. The process pipeline implements a highly transparent and portable model of job control and monitoring where different assembly stages can be executed and re-executed separately or in unison on a wide variety of architectures.
+    license: Open Source
```
Comment (bgruening) [inline recipes/meraculous/meta.yaml]: can this be more specific?

**Comment by ysu20** [comment]
Thanks for the review @bgruening I have submitted a commit in response to your comments.

File: recipes/meraculous/meta.yaml (hunk 1 above)
Comment (ysu20) [reply recipes/meraculous/meta.yaml]: This is now removed by my second commit.

File: recipes/meraculous/meta.yaml (hunk 2 above)
Comment (ysu20) [reply recipes/meraculous/meta.yaml]: This is now specified in my second commit.

**Comment by bgruening** [comment]
@ysu20 thanks, looks great!

**Pull request merged**